uniffi_bindgen/src/bindings/swift/test.rs

```diff
@@ -153,25 +151,9 @@ impl GeneratedSources {
         let main_module = main_source.config.module_name();
         let modulemap_glob = glob(&out_dir.join("*.modulemap"))?;
         let module_map = match modulemap_glob.len() {
-            0 => bail!("No modulemap files found in {out_dir}"),
-            // Normally we only generate 1 module map and can return it directly
+            // write_bindings should have generated exactly 1 module map
             1 => modulemap_glob.into_iter().next().unwrap(),
-            // When we use multiple UDL files in a test, for example the ext-types fixture,
-            // then we get multiple module maps and need to combine them
-            _ => {
-                let path = out_dir.join("combined.modulemap");
-                let mut f = File::create(&path)?;
-                write!(
-                    f,
-                    "{}",
-                    modulemap_glob
-                        .into_iter()
-                        .map(|path| Ok(read_to_string(path)?))
-                        .collect::<Result<Vec<String>>>()?
-                        .join("\n")
-                )?;
-                path
-            }
+            n => bail!("{n} modulemap files found in {out_dir}"),
```

Comment:
I'm not a Swift expert, but my hope is that if the tests pass after this change then the single-modulemap should also work for real-world consumption.